lib/web_ui/test/golden_tests/engine/screenshot.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'dart:html' as html;
+import 'package:ui/ui.dart' as ui;
+import 'package:ui/src/engine.dart';
+import 'package:web_engine_tester/golden_tester.dart';
+import 'package:test/test.dart';
+
+// Commit a recording canvas to a bitmap, and compare with the expected
```

Comment:
super nit :)

doc comments "///". Period after the comment.

Reply:
Done. Thank you!